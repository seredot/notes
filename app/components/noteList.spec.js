import "react-native";
import React from "react";
import NoteList from "./noteList";

import renderer from "react-test-renderer";

describe("Testing NoteList component", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<NoteList />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
