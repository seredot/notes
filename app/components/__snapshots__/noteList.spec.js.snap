// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NoteList component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F2EB61",
      "flex": 1,
      "justifyContent": "center",
      "paddingTop": 4,
    }
  }
>
  <RCTScrollView
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "alignSelf": "stretch",
        "flex": 1,
        "paddingBottom": 8,
      }
    }
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
